.github/workflows/build-test-actions.yml

```diff
@@ -5,33 +5,38 @@ permissions:
 
 on:
   pull_request:
-  # https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request
+    # https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request
     types: [opened, reopened, converted_to_draft, ready_for_review, synchronize]
     branches:
       - master
   push:
     branches:
-    # https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-workflow-only-when-a-push-to-specific-branches-occurs
-        - master
+      # https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-workflow-only-when-a-push-to-specific-branches-occurs
+      - master
 
 jobs:
   code-style:
     strategy:
       fail-fast: false
       matrix:
-          # Add versions here to expand support matrix
+        # Add versions here to expand support matrix
         python-version: ["3.10", "3.11", "3.12"]
         os: [ubuntu-latest, macos-latest]
     runs-on: ${{ matrix.os }}
     env:
       PYTHON: ${{ matrix.python-version }}
     steps:
       - uses: actions/checkout@v4
+      - name: Get Ruff version from pyproject.toml
+        id: get-ruff-version
+        run: |
+          RUFF_VERSION=$(grep 'ruff = ' pyproject.toml | cut -d'"' -f2)
```

Comment:
Does this only search for the line below? Just wanted to check because I don't know what `-f2` does. 

https://github.com/vprusso/toqito/blob/1df1159b1def17afdf7eae611cf949011830dc7b/pyproject.toml#L58

`ruff` is mentioned more than once in the `toml` file.

Reply:
it might be worth adding a couple of comments about how your added lines work. These are useful if someone else decides to make changes to these jobs in the future.